sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -68,6 +68,11 @@ def _get_list_element_type(typ: Optional[type]) -> Optional[type]:
     if typ is None:
         return None
 
+    # Annotations not specifying the element type are assumed by mypy
+    # to signify Any element type. Follow suit here.
+    if typ in {list, List, Sequence, abc.Sequence}:
+        return cast(type, Any)
```

Comment:
Based on the usage, and how we do similar checks for dicts, we could return `None` here. But returning `Any` also works (since all the places that use this will convert `Any` to `None` and aligning with mypy makes sense to me.